homeassistant/components/slide_local/diagnostics.py

```diff
@@ -0,0 +1,27 @@
+"""Provides diagnostics for slide_local."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.const import CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+
+from . import SlideConfigEntry
+
+TO_REDACT = [
+    CONF_PASSWORD,
+]
```

Comment:
Hah, at this point I would even wonder if we consider sensitive given that the manufacturer openly published them 😂